nvtabular/workflow.py

```diff
@@ -508,6 +510,10 @@ def apply_ops(self, gdf, start_phase=None, end_phase=None, writer=None, output_p
                     writer.need_cal_col_names = False
 
                 start_write = time.time()
+                # Special dtype conversion
+                if dtypes:
+                    _meta = _set_dtypes(gdf._meta, dtypes)
+                    gdf = gdf.map_partitions(_set_dtypes, dtypes, meta=_meta)
```

Comment:
I believe `apply_ops` is used for online preprocessing, where you are not actually dealing with a `dask_cudf.DataFrame` object.  I *think* this should probably be as simple as `gdf = _set_dtypes(dtypes)`

Reply:
Sorry @albert17 ! ! Typo on my end.  I meant to suggest `gdf = _set_dtypes(gdf, dtypes)`

Reply:
Got it! Thanks!